github/util.py

```diff
@@ -659,6 +666,10 @@ def add_labels_to_pull_request(self, pull_request_number, *labels):
         pull_request = self.repository.pull_request(pull_request_number)
         pull_request.issue().add_labels(*labels)
 
+    def remove_label_from_pull_request(self, pull_request_number, label):
+        pull_request = self.repository.pull_request(pull_request_number)
+        pull_request.issue().remove_label(label)
```

Comment:
inline (unless used more than once)

otherwise, maybe mv as util-function to github/util.py?